src/panoramic/cli/transform/compiler.py

```diff
@@ -0,0 +1,21 @@
+from requests import RequestException
+
+from panoramic.cli.errors import TransformCompileException
+from panoramic.cli.transform.client import TransformClient
+from panoramic.cli.transform.pano_transform import CompiledTransform, PanoTransform
+
+
+class TransformCompiler:
+    @classmethod
+    def compile(cls, transform: PanoTransform, company_slug: str) -> CompiledTransform:
+        try:
+            transform_client = TransformClient()
```

Comment:
Would be good to reuse the client between calls, otherwise this is fetching token each time...

Reply:
Addressed in 9da198e